pkg/daemon/config.go

```diff
@@ -90,8 +107,8 @@ func (l *LinuxPTPConfUpdate) UpdateConfig(nodeProfilesJson []byte) error {
 }
 
 // Try to load the multiple policy config
-func tryToLoadConfig(nodeProfilesJson []byte) ([]ptpv1.PtpProfile, bool) {
-	ptpConfig := []ptpv1.PtpProfile{}
+func tryToLoadConfig(nodeProfilesJson []byte) (nProfiles, bool) {
+	ptpConfig := nProfiles{}
```

Comment:
I think its better we use same 
type in 4.16 also

Reply:
why not use same sorting method in 4.16 too @josephdrichard . So that we don't have too much code variation